tests/tas-integration/pom.xml

```diff
@@ -114,8 +114,10 @@
                     <suiteXmlFiles>
                         <suiteXmlFile>${suiteXmlFile}</suiteXmlFile>
                     </suiteXmlFiles>
+                    <!-- Keeping illegal-access=warn for Java 11 compatibility, even though it has no effect on JDK 17 -->
```

Comment:
Can it be removed when we switch back to Java11?

Reply:
yes